### PR TITLE
FOGL 637/640/643 - Provide "ALIAS" support in Payload builder + json type support + TODO removal

### PR DESCRIPTION
**Pull request by amarendra-dianomic**
Test suite result:
`=================================================== 1086 passed, 13 skipped in 48.10 seconds ====================================================
`

### CODE REVIEW

File: python/foglamp/common/storage_client/payload_builder.py [hunk 1]
```diff
@@ -90,8 +101,153 @@ def is_json(myjson):
...
+        :param main_key: either "return" or "aggregate"
+        :param args: each arg is a tuple. The len of tuple depends upon main_key. If main_key is "return" i.e. SELECT,
+                     then each tuple will contain (col, alias). If main_key is "aggregate", then each tuple will contain
+                     (col, operation, alias) because col can be repeated in aggregate with different "operations".
+        :return:
+        :example:
+        PayloadBuilder().SELECT(("name", "id")).ALIAS('return', ('name', 'my_name'), ('id', 'my_id')).payload() prints
+            {"return": [{"column": "name", "alias": "my_name"},
+                        {"column": "id", "alias": "my_id"}]}
+
```
Comment (Singhal-Vaibhav) [inline python/foglamp/common/storage_client/payload_builder.py]: Can we have a better docstring for this internal function? We are using the same docstring and example of ALIAS here

**Comment by amarendra-dianomic** [comment]
@ashish-jabble code fixed. All tests now pass. Pl check.

File: tests/unit/python/foglamp/common/storage_client/test_payload_builder.py [hunk 2]
```diff
@@ -155,6 +155,28 @@ def test_group_by_payload(self, test_input, expected):
         res = PayloadBuilder().GROUP_BY(test_input).payload()
         assert expected == json.loads(res)
 
+    @pytest.mark.parametrize("test_input, expected", [
+        ('user_ts', _payload("data/payload_group_by2_alias.json"))
```
Comment (ashish-jabble) [inline tests/unit/python/foglamp/common/storage_client/test_payload_builder.py]: file missing

File: tests/unit/python/foglamp/common/storage_client/test_payload_builder.py [hunk 3]
```diff
@@ -155,6 +155,28 @@ def test_group_by_payload(self, test_input, expected):
...
+    ])
+    def test_group_by_payload_alias1(self, test_input, expected):
+        res = PayloadBuilder().GROUP_BY(test_input).ALIAS('group', ('user_ts', 'timestamp')).payload()
+        assert expected == json.loads(res)
+
+    @pytest.mark.parametrize("test_input, expected", [
+        ("user_ts", _payload("data/payload_group_by1_alias.json"))
```
Comment (ashish-jabble) [inline tests/unit/python/foglamp/common/storage_client/test_payload_builder.py]: file missing

**Comment by amarendra-dianomic** [comment]
@ashish-jabble added missing files. Pl check.

**Comment by ashish-jabble** [comment]
@amarendra-dianomic It would be better if we provide the support for `json_properties` payload to the payload of the update entry point
```
"json_properties" : [
				{
					"column" : "data",
					"path"   : [ "url", "value" ],
					"value"  : "new value"
				}
			    ]
```

**Pull request merged**